### PR TITLE
WFDR-104.2-EditPrefs

### PR DESCRIPTION
**Pull request by danielqyu**
## Summary of changes
Create edit prefs page for existing users

## Trello Task link
https://trello.com/c/NtZ91q5q/18-1042-preference-page-editing-for-existing-users

## Corresponding User Story Number and Title
User story 4: Edit preferences

## Detailed list of changes
File found in templates/user/editPrefs.html
Based on addPrefsNewUsers.html, changes from that are as follows:
- added back button to return to home
- add thymeleaf in each preference to pull from the current user attributes
- attributes pulled are: user.access, user.diet, user.language1, user.language2, and user.language3

Needs to be tested with the database, and redirects need to be added

### CODE REVIEW

**Review by setaDM — APPROVED** [review]
Looks good. I'll make sure to add the endpoints to redirect the user to the login page and acc settings.

File: src/main/resources/templates/user/editPrefs.html [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+    <!--Add prefs container-->
+    <div class="container-fluid d-flex justify-content-center">
+        
+        <div class="col-md-4 m-sm-4" id="formInputsCont">
+            <!--ADD CORRECT FORM PATH LATER-->
+            <form action="/" method="post" class="errorCheck" novalidate>
+                <h2 class="row justify-content-center">Edit Preferences</h2>
+                <div class="container p-1">
+                    <label for="access">Accessibility</label>
+                    <select class="form-select" id="access" name="access" required th:value="${user.access}">
```
Comment (NickMoo5) [inline src/main/resources/templates/user/editPrefs.html]: This won't select values that you want to be selected,please see my as2 for reference

File: src/main/resources/templates/user/editPrefs.html [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+                        <option selected disabled value="">Choose...</option>
+                        <option>Wheelchair</option>
+                        <option>None</option>
+                    </select>
+                    <div class="invalid-feedback">Please select an option</div>
+                </div>
+                <div class="container p-1">
+                    <label for="diet">Dietary Restrictions</label>
+                    <select class="form-select" id="diet" name="diet" required th:value="${user.diet}">
```
Comment (NickMoo5) [inline src/main/resources/templates/user/editPrefs.html]: Same as above

File: src/main/resources/templates/user/editPrefs.html [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+                        <option>Vegetarian</option>
+                        <option>Vegan</option>
+                        <option>Pescatarian</option>
+                        <option>None</option>
+                    </select>
+                    <div class="invalid-feedback">Please select an option</div>
+                </div>
+                <div class="container p-1">
+                    <label for="language1">Language Preferred #1</label>
+                    <select class="form-select" id="language1" name="language1" required th:value="${user.language1}">
```
Comment (NickMoo5) [inline src/main/resources/templates/user/editPrefs.html]: Same as above

File: src/main/resources/templates/user/editPrefs.html [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+                        <option>English</option>
+                        <option>French</option>
+                        <option>Spanish</option>
+                    </select>
+                    <div class="invalid-feedback">Please select an option</div>
+                </div>
+                <div class="container p-1">
+                    <label for="language2">Language Preferred #2</label>
+                    <select class="form-select" id="language2" name="language2" required th:value="${user.language2}">
+                        <option selected disabled value="">Choose...</option>
```
Comment (NickMoo5) [inline src/main/resources/templates/user/editPrefs.html]: Same as above

File: src/main/resources/templates/user/editPrefs.html [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+                        <option>English</option>
+                        <option>French</option>
+                        <option>Spanish</option>
+                        <option>None</option>
+                    </select>
+                    <div class="invalid-feedback">Please select an option</div>
+                </div>
+                <div class="container p-1">
+                    <label for="language3">Language Preferred #3</label>
+                    <select class="form-select" id="language3" name="language3" required th:value="${user.language3}">
```
Comment (NickMoo5) [inline src/main/resources/templates/user/editPrefs.html]: Same as above

**Review by NickMoo5 — CHANGES_REQUESTED** [review]
Please look at my as2 and update the code :))

**Review by NickMoo5 — APPROVED** [review]
Merge with devMain and it's good

**Pull request merged**